### PR TITLE
feat(apigatewayv2): add `description` property for stage

### PR DESCRIPTION
**Pull request by yoyoyo-pg**
### Reason for this change

The `description` property existed in the L1 construct but was not present in the L2 construct

### Description of changes

Add the `description`  property for `HttpStage` and `WebSocketStage`, which was missing in the L2 construct.

### Description of how you validated changes

I Added both unit test and integration tests.

### Checklist
- [x] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md) and [DESIGN GUIDELINES](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by yoyoyo-pg** [comment]
Exemption Request: It is possible to specify the `description` option when defining `CfnApi` as well, but there is no description in the apigatewayv2 README.
I don't think it is necessary to include a description for `CfnStage` as well as for `CfnApi`.

**Review by go-to-k — CHANGES_REQUESTED** [review]
Thanks for this PR! It seems that the PR title has a typo `poperty`.
 There are also errors in the Code Build CI. Could you please check the trailing spaces, and erase them?

```
aws-cdk-lib: /codebuild/output/src284929355/src/github.com/aws/aws-cdk/packages/aws-cdk-lib/aws-apigatewayv2/lib/common/stage.ts
aws-cdk-lib:   72:5  error  Trailing spaces not allowed  no-trailing-spaces
aws-cdk-lib: /codebuild/output/src284929355/src/github.com/aws/aws-cdk/packages/aws-cdk-lib/aws-apigatewayv2/test/websocket/stage.test.ts
aws-cdk-lib:   148:1  error  Trailing spaces not allowed  no-trailing-spaces
aws-cdk-lib: 笨� 2 problems (2 errors, 0 warnings)
aws-cdk-lib:   2 errors and 0 warnings 
```

**Comment by yoyoyo-pg** [comment]
@go-to-k 
Thank you! I have modified the title of the pull request and removed the trailing space.

File: packages/aws-cdk-lib/aws-apigatewayv2/lib/common/stage.ts [hunk 1]
```diff
@@ -66,6 +66,13 @@ export interface StageOptions {
    * @default - no throttling configuration
    */
   readonly throttle?: ThrottleSettings;
+
+  /**
+   * The description for the api stage
+   *
+   * @default - none
```
Comment (go-to-k) [inline packages/aws-cdk-lib/aws-apigatewayv2/lib/common/stage.ts:73]: nit (Docs from https://docs.aws.amazon.com/ja_jp/AWSCloudFormation/latest/UserGuide/aws-resource-apigatewayv2-stage.html#cfn-apigatewayv2-stage-description)

```suggestion
   * The description for the API stage
   *
   * @default - no description
```

**Review by go-to-k — CHANGES_REQUESTED** [review]
It's very nice. I made just a few minor comments.

**Comment by go-to-k** [comment]
> Exemption Request: It is possible to specify the description option when defining CfnApi as well, but there is no description in the apigatewayv2 README.
I don't think it is necessary to include a description for CfnStage as well as for CfnApi.

Both the WebSocketStage and the HttpStage seem to be in the README. Could you include the properties in the sample code?

https://github.com/go-to-k/aws-cdk/blob/7cd0f65a669abf5144a705fec0652f3b550a9340/packages/aws-cdk-lib/aws-apigatewayv2/README.md#publishing-http-apis

https://github.com/go-to-k/aws-cdk/blob/7cd0f65a669abf5144a705fec0652f3b550a9340/packages/aws-cdk-lib/aws-apigatewayv2/README.md#websocket-api

```ts
declare const api: apigwv2.HttpApi;

new apigwv2.HttpStage(this, 'Stage', {
  httpApi: api,
  stageName: 'beta',
  description: 'My Stage',
});
```

**Review by go-to-k — APPROVED** [review]
Thanks. LGTM.

**Review by gracelu0 — APPROVED** [review]
LGTM, thank you for contributing!

**Review by gracelu0 — APPROVED** [review]

**Pull request merged**